gc/base/GCExtensionsBase.hpp

```diff
@@ -1124,6 +1129,11 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		return objectModel.getObjectAlignmentInBytes();
 	}
 
+	MMINLINE float
+	getDarkMatterCompactThreshold(){
```

Comment:
Can you put a space between `()` and the `{`?

Reply:
NVM about this unless @amicic has other changes that he would like to see added.